ReachCare/core/fixtures/no_site_addresses.json

```diff
@@ -0,0 +1 @@
1, "site_name": "Compass Community Health", "site_phone": "(740) 355-7102"}}, {"model": "core.testingsite", "pk": 6272, "fields": {"provider": 536, "address": 5757, "site_name": "Open Door Family Planning Clinic", "site_phone": "(765) 286-7000"}}, {"model": "core.testingsite", "pk": 6273, "fields": {"provider": 517, "address": 5768, "site_name": "Meridian Health Services Corp", "site_phone": "(765) 641-7499"}}, {"model": "core.testingsite", "pk": 6274, "fields": {"provider": 517, "address": 5771, "site_name": "Meridian Health Services Corp", "site_phone": "(765) 552-0841"}}, {"model": "core.testingsite", "pk": 6275, "fields": {"provider": 517, "address": 5775, "site_name": "Meridian Health Services Corp", "site_phone": "(765) 442-0570"}}, {"model": "core.testingsite", "pk": 6276, "fields": {"provider": 692, "address": 5800, "site_name": "Marram Health Center", "site_phone": "(219) 806-3000"}}, {"model": "core.testingsite", "pk": 6277, "fields": {"provider": 549, "address": 5822, "site_name": "Great Lakes Bay Health Centers Wolverine Vassar", "site_phone": "(989) 823-3040"}}, {"model": "core.testingsite", "pk": 6278, "fields": {"provider": 552, "address": 5832, "site_name": "Grace Health, Inc.", "site_phone": "(517) 629-6540"}}, {"model": "core.testingsite", "pk": 6279, "fields": {"provider": 549, "address": 5866, "site_name": "Great Lakes Bay Health Centers Women&#39;s Care West Branch", "site_phone": "(989) 345-0100"}}, {"model": "core.testingsite", "pk": 6280, "fields": {"provider": 560, "address": 5895, "site_name": "ALIVIO MEDICAL CENTER", "site_phone": "(773) 254-1400"}}, {"model": "core.testingsite", "pk": 6281, "fields": {"provider": 566, "address": 5921, "site_name": "Howard Brown Health 55th Street", "site_phone": "(773) 388-1600"}}, {"model": "core.testingsite", "pk": 6282, "fields": {"provider": 492, "address": 5991, "site_name": "COMMUNITY HEALTH PARTNERSHIP OF ILLINOIS", "site_phone": "(630) 859-0015"}}, {"model": "core.testingsite", "pk": 6283, "fields": {"provider": 576, "address": 5996, "site_name": "Kenosha Community Health Center, Inc.", "site_phone": "(262) 764-3605"}}, {"model": "core.testingsite", "pk": 6284, "fields": {"provider": 576, "address": 5997, "site_name": "Kenosha Community Health Center, Inc.", "site_phone": "(262) 656-0044"}}, {"model": "core.testingsite", "pk": 6285, "fields": {"provider": 576, "address": 5998, "site_name": "Kenosha Community Health Center, Inc.", "site_phone": "(262) 656-0044"}}, {"model": "core.testingsite", "pk": 6286, "fields": {"provider": 595, "address": 6071, "site_name": "COOS COUNTY FAMILY HEALTH SERVICES", "site_phone": "(603) 466-2741"}}, {"model": "core.testingsite", "pk": 6287, "fields": {"provider": 603, "address": 6084, "site_name": "LAMPREY HEALTH CARE", "site_phone": "(603) 659-2494"}}, {"model": "core.testingsite", "pk": 6288, "fields": {"provider": 604, "address": 6101, "site_name": "Program for All-Inclusive Care for the Elderly (PACE) Site", "site_phone": "(978) 712-3360"}}]
```

Comment:
What's this for? Do I need to load this when we deploy?

Reply:
When I built the fixture in json, I dropped any testingsite with a duplicated name, even if it was two different addresses on the excel spreadsheet.  This meant that there were 226 addresses in the database with no testingsites.   When using the next feature, it creates an error "No testingsite in your area",  when there actually is a testingsite, it just isn't in the database. 

yes, you need to load this when we deploy.

Reply:
Ok, cool, and to clarify, I need to drop the table and reload? or just add this one?

Reply:
just add this one. it should add 226 records to the testingsite table.